src/lobby.js

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import mj from "minijanus";
+
+import Config from "./config";
+
+import "./lobby.css";
+
+class Lobby extends React.Component {
+  constructor() {
+    super();
+    this.state = { rooms: [] };
+
+    this.onWebsocketOpen = this.onWebsocketOpen.bind(this);
+    this.onWebsocketMessage = this.onWebsocketMessage.bind(this);
+    this.updateRooms = this.updateRooms.bind(this);
+  }
+
+  componentDidMount() {
+    this.ws = new WebSocket(Config.janus_server_url, "janus-protocol");
+    this.session = new mj.JanusSession(this.ws.send.bind(this.ws));
+    this.ws.addEventListener("open", this.onWebsocketOpen);
+    this.ws.addEventListener("message", this.onWebsocketMessage);
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.pollRoomsIntervalHandle);
+    this.ws.removeEventListener("open", this.onWebsocketOpen);
+    this.ws.removeEventListener("message", this.onWebsocketMessage);
+  }
+
+  onWebsocketOpen() {
+    this.session
+      .create()
+      .then(() => {
+        this.handle = new mj.JanusPluginHandle(this.session);
+        return this.handle.attach("janus.plugin.sfu").then(this.updateRooms);
+      })
+      .then(() => {
+        this.pollRoomsIntervalHandle = setInterval(this.updateRooms, 10000);
+      });
+  }
+
+  updateRooms() {
+    return this.fetchRooms().then(rooms => this.setState({ rooms }));
+  }
+
+  fetchRooms() {
+    return Promise.all(
+      Config.public_rooms.map(room_id => {
+        return this.handle
+          .sendMessage({
+            kind: "listusers",
+            room_id
+          })
+          .then(signal => ({
+            id: room_id,
+            limit: 12,
```

Comment:
Is this limit just for show right now?

Reply:
Yep, just imagining how that might work. May just remove it.